lib/colorls/flags.rb

```diff
@@ -23,38 +23,39 @@ def initialize(*args)
 
     def process
       incompatible = report_incompatible_flags
-      return STDERR.puts "\n   #{incompatible}".colorize(:red) if incompatible
+      STDERR.puts "\n   #{incompatible}".colorize(:red) if incompatible
```

Comment:
I think this `return` needs to stay, to stop the program from going to the below if-else when there are incompatible tags.